### PR TITLE
Use Typography.js to setup site typography

### PR DESCRIPTION
**Pull request by KyleAMathews**
screen shot 2016-08-02 at 12 13 51 pm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,16 @@
...
 
+### CSS
+We use Sass + [Typography.js](https://github.com/KyleAMathews/typography.js)
+
+To setup Typography.js, go to the scripts folder and run `npm install`.
+
+Then to make changes to the generated typography css, make changes to
+`write-typography.js` and then run `npm run write-typography`. This will
+write out a file to the \_sass src directory. Then just rebuild main.css
+by running from the root directory `sass css/main.scss --load-path _sass/ > css/main.css`
```
Comment (mxstbr) [inline README.md]: Jekyll actually builds the Sass automatically, no need to manually compile it. :+1:

**Comment by mxstbr** [comment]
Hmm, for some reason I'm not getting any styles when using this branch and the `sass` command doesn't work either…

**Comment by KyleAMathews** [comment]
Yeah.... Ruby/Gems/Bundler always manage to confuse me. I had assumed Jekyll would handle compiling styles too and was surprised that wasn't working. I think I have Sass installed globally if that matters at `/usr/bin/sass`.

**Comment by mxstbr** [comment]
Ah I know why, you need to leave the 

```
----
----
```

in the main.scss file!

**Comment by mxstbr** [comment]
I've got it working and oi, this is so much nicer! Ace job dude, this looks great!

The difference between the heading levels is a bit too small imo, but let's merge this for now and deal with that in a followup PR!

**Comment by mxstbr** [comment]
blockquotes also seem really big, I use them Github style to annotate things with more, but not 100% necessary information like this:

Some information

> Some not necessary background information

screen shot 2016-08-04 at 10 48 39

**Pull request merged**